src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+
+  if (days < 1) {
+    return 0;
+  } else if (days >= LONG_TERM) {
+    return days * 40 - LONG_TERM_DISCOUNT;
+  } else if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
try to create a  "let" variable and update your result in here instead of returning

Reply:
There is nothing wrong with returning here.